app/src/main/java/com/garethevans/church/opensongtablet/StageMode.java

```diff
@@ -1921,8 +1990,18 @@ private void setupPageButtons() {
                 pageButtonAlpha("");
             }
         });
+        // Button groups - extra group long click action to switch to all group mode (collapse of the buttons displayed)
+        extraButton.setOnLongClickListener(view -> {
+            CustomAnimations.animateFAB(extraButton,StageMode.this);
+            preferences.setMyPreferenceBoolean(StageMode.this, "pageButtonGroupMain", !(preferences.getMyPreferenceBoolean(StageMode.this, "pageButtonGroupMain", false)));
+            // Need to redo OnClick button action
+            setupPageButtons();
+            groupPageButtons();
+            // Make sure all dynamic (scroll and set) buttons display
+            onScrollAction();
+            return true;
+        });
```

Comment:
Yes, I like this and see what you did on the PopUpPageButtonsFragment that I queried.

Reply:
OK